sdk/servicebus/Azure.Messaging.ServiceBus/tests/Transactions/TransactionLiveTests.cs

```diff
@@ -417,5 +417,322 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
                 Assert.Null(receivedMessage);
             }
         }
+
+        [Test]
+        [TestCase(true, true)]
+        [TestCase(true, false)]
+        [TestCase(false, false)]
+        [TestCase(false, true)]
+        public async Task TransactionGroupReceivesFirst(bool partitioned, bool enableSessions)
```

Comment:
Should we consider splitting out the session vs. non-session version of this?   It seems the logic and flow differ enough that it may be helpful.